### DISCUSSION TITLE
Project Submission

### DISCUSSION THREAD

**Discussion by ichko** [ichko/fmi-fp-2020-21#11] [Show and tell]
Моля след като си изберете проект от описаните [тук](https://docs.google.com/document/d/1nfYOuSN5ss3LKlO60XA2NiUbTV4Dmpsj0zCNxp-MfZ4/edit) (или избран от вас) моля:

1. Направете си `private` GitHub репо на проекта.
2. Добавете @ichko, @svilenbs, @semerdzhiev към contributor-ите.
3. Добавете линк към прайвът репото си в това `issue` заедно със заглавие на проекта и име.

**Comment by ichko** [comment]
Пример:

> Haskell - RayTracer - Iliya Zhechev 88125
    https://github.com/ichko/fmi-fp-2020-21

**Comment by hspasov** [comment]
> Haskell - NeuralNetworks - Hristo Spasov 62278
> https://github.com/hspasov/NeuralNetworks

**Comment by hvtonev** [comment]
> Haskell - BattleRoyalSnakeGame - Hristiyan Tonev 45521
> https://github.com/hvtonev/haskell-snake-royale

**Comment by DNaydenov** [comment]
Haskell - ChessBot - Димитър Найденов 62185 
https://github.com/DNaydenov/chess-bot

**Reply by googleson78** [reply]
Този `ChessBot` сега по-лесен ли ще се окаже от интерпретатор за Scheme?

**Reply by googleson78** [reply]
:P

**Reply by DNaydenov** [reply]
Хаха, не е задължително да го предавам, нали така 😆

**Comment by nzaharov** [comment]
Haskell - Raytracer - Николай Захаров 62149
https://github.com/nzaharov/raytracer-hs

**Comment by GeorgiAnchev** [comment]
NeuralNetwork - Georgi Anchev 61816
https://github.com/GeorgiAnchev/NeuralNetworkInHaskell

**Reply by GeorgiAnchev** [reply]
Става ли да ползвам същия проект и за основния предмет?

**Reply by ichko** [reply]
Няма проблем, ползвай го за каквото искаш, твой си е! :relieved:

**Comment by kathiekk** [comment]
Haskell - Huffman Coding - Katerina Koleva
https://github.com/kathiekk/Huffman-Coding.git

**Comment by dimitrov-k** [comment]
Haskell - RayTracer - Kristian DImitrov  - 62260 - Miroslav Stefanov - 62282
https://github.com/MiroslavStefanov/RayTracer

**Reply by svilenbs** [reply]
Error 404 Not found.

**Reply by ichko** [reply]
@dimitrov-k Може ли да ни добавиш като contributors към репо-то?

**Comment by AIK-dev** [comment]
RayTracer - Alexander Karaivanov 45449
https://github.com/AIK-dev/RayTracer-Haskell

**Comment by IoanStoianov** [comment]
Haskell - Neural Network - Ioan Stoianov 81596

https://github.com/IoanStoianov/NeuralNetwork
